modules/swagger-codegen/src/main/java/io/swagger/codegen/CodegenOperation.java

```diff
@@ -117,7 +117,7 @@ public boolean isRestfulIndex() {
      * @return true if act as Restful show method, false otherwise
      */
     public boolean isRestfulShow() {
-        return "GET".equals(httpMethod) && isMemberPath();
```

Comment:
@sebymiano 

> I'm sorry, it was my fault.

No worry. We appreciate your contribution for another C++ server generator.

> FYI, I added equalsIgnoreCase rather than equals because the httpMethod variable is not always uppercase. Do you agree?

Yup, make sense to me.